examples/fifth/main.go

```diff
@@ -51,8 +51,9 @@ func init() {
 func main() {
 	name := "fifth"
 	version := "1.0.0"
+	env := "qaco"
```

Comment:
As Patron is a public-facing project, *qaco* makes little sense.

We could change it to eg. *staging* or *demo*, so it's more evident what it stands for.